src/program/snabbvmx/README.md

```diff
@@ -0,0 +1,107 @@
+### SnabbVMX
+
+SnabbVMX is a network design that combines Snabb's lwAFTR app and a VM managed by Snabb.
+
+Snabb's lwAFTR app, in addition to managing lw4o6 traffic (IPv4 and IPv6 packets),
+it also manages non-lwAFTR traffic such as pings to lwAFTR interfaces, ARP and
+NDP resolution.  This traffic is managed by addition applications connected to
+the lwAFTR network design in program/lwaftr/run.lua.  Management of this type
+of traffic is a requiremennt in RFC 7596.
+
+SnabbVMX uses a different approach.  It setups a network design where only lw4o6
+packets are managed by the lwAFTR app, while everything else is forwarded to a VM
+which is in charge of providing ping, ARP and NDP resolution.
+
+#### App network
+
+SnabbVMX's app network:
+
+![SnabbVMX](.images/snabbvmx.png)
```

Comment:
Fixed now. It seems that due to a .gitignore rule in `/.gitignore`, folders called `.images` are not tracked. Must `git add -f <file>` to add them. Reminder for the future :)